SS2-Project/Assets/Starstorm2/Modules/Characters/Monsters/Mimic.cs

```diff
@@ -175,15 +170,7 @@ private void RebuildPingOverrideInteractable(ILContext il)
 				return;
 			}
 
-			// And turn it into
-			// if (displayNameProvider != null)
-			// {
-			//     if (pingTarget.TryGetComponent<MimicController>(out var controller))
-			//     {
-			//         DoOurStuff();
-			//     }
-			//     else DoStuff();
-			// }
```

Comment:
Why are we removing ILHook documentation? isnt this something you requested?? am i missing something?

Reply:
ok added it all back good catch